utils/rpms/daos.spec

```diff
@@ -305,7 +305,7 @@ mkdir -p %{buildroot}/%{_unitdir}
 %if (0%{?rhel} == 7)
 install -m 644 utils/systemd/%{server_svc_name}.pre230 %{buildroot}/%{_unitdir}/%{server_svc_name}
 install -m 644 utils/systemd/%{agent_svc_name}.pre230 %{buildroot}/%{_unitdir}/%{agent_svc_name}
-%else
+%else if (0%{?rhel} == 8 or  %{?suse_version} == 1500)
```

Comment:
Isn't the addition of the `if ...` here the implied condition without it?

IOW, given that we only build on EL7, EL8 and LEAP15, even without the addition here, the logic would be the same wouldn't it?

In any case I don't think `%else if` is valid syntax (happy to be proven wrong if you can find some docs on it).  What you would be looking for I think is `%elif` added in https://github.com/rpm-software-management/rpm/commit/1c4b238840d0995344d0d0381a0561b213429203 but according to the git tags on that commit, that is only available from RPM version 4.15+ and EL7 still only has 4.11, so `%elif` is not available to us.

Reply:
During submission of this PR I did try both else if and elif. I don't recall the exact error, but as expected when trying elif rpmbuild would not complete. Then with using "else if" the rpmbuild completes without. Each distro was then tested to ensure the correct file was in place. In hindsight, it must have just parsed the "else" portion and ignored "if (0%{?rhel}...". 

So in summary, while the syntax is incorrect, the functionality is working as expected.